trestle/core/resolver/_import.py

```diff
@@ -81,6 +88,19 @@ def __init__(
                     f'Back matter resource resolution needed for profile import failed with error: {str(e)}'
                 )
 
+        uri_type = cache.FetcherFactory.get_uri_type(self._import.href)
+        # if this looks like a relative path to remote source, append parent path
+        if uri_type == cache.FetcherFactory.UriType.LOCAL_FILE and self._parent_url_root:
+            self._import.href = self._parent_url_root + self._import.href
+        # if href is now a remote path, capture its parent path for possible use with child imports that are relative
+        if uri_type not in [cache.FetcherFactory.UriType.LOCAL_FILE, cache.FetcherFactory.UriType.TRESTLE]:
+            head_path, _ = os.path.split(self._import.href)
+            if head_path and head_path[-1] != '/':
+                head_path += '/'
```

Comment:
Use `os.path.dirname(self._import.href)` and then append two paths so you don't have to manually add `\` at the end

Reply:
I have been burned by trusting some of these path functions, including .stem, that don't work as expected.  So when it is very simple and clear I prefer an explicit operation.  But I will make the change to use os here.